src/components/Tooltip/placements.ts

```diff
@@ -0,0 +1,13 @@
+import { Placement } from "@popperjs/core"
+
+const basePlacement = ["top", "bottom", "left", "right", "auto"]
+const startPlacement = basePlacement.map((placement) => `${placement}-start`)
+const endPlacement = basePlacement.map((placement) => `${placement}-end`)
+
+// while @popperjs/core has its own placements array, in some cases the export is undefined, specifically within a testing environment
+
+export const placements = [
+  ...basePlacement,
+  ...startPlacement,
+  ...endPlacement,
+] as Placement[]
```

Comment:
This is minor, but I'd suggest using a type annotation rather than an assertion (`as`) so that we can be sure that our assignment really does conform to `Placement[]`. As it is, if you change `...-end` to `...-eeend` on line 5, no error is raised.

```suggestion
import { Placement } from "@popperjs/core"

const basePlacement = ["top", "bottom", "left", "right", "auto"] as const
const startPlacement = basePlacement.map((placement) => `${placement}-start` as const)
const endPlacement = basePlacement.map((placement) => `${placement}-end` as const)

// while @popperjs/core has its own placements array, in some cases the export is undefined, specifically within a testing environment

export const placements: Placement[] = [
  ...basePlacement,
  ...startPlacement,
  ...endPlacement,
]
```

Reply:
Nice, good suggestion.

Reply:
I didn't realize you could use const within the map itself so that's super handy.